electron/utils.ts

```diff
@@ -337,6 +344,37 @@ function showItemInFolder(item: string) {
   }
 }
 
+function splitPathAndName(fullPath: string): { dir: string; bin: string } {
+  const dir = dirname(fullPath)
+  let bin = basename(fullPath)
+  // On Windows, you can just launch executables that are in the current working directory
+  // On Linux, you have to add a ./
+  if (!isWindows) {
+    bin = './' + bin
+  }
+  return { dir: dir, bin: bin }
```

Comment:
if key and value have the same name, you can just do it like this:
`return {dir, bin}`

ofc the way you are doing its not wrong as well 👍🏽